scripts/west_commands/ncs_commands.py

```diff
@@ -212,13 +213,16 @@ def print_loot(self, name, project, z_project, args):
         else:
             z_rev = z_project.revision
 
+        n_rev = 'refs/heads/manifest-rev'
+
         try:
-            nsha = project.sha(project.revision)
+            nsha = project.sha(n_rev)
             project.git('cat-file -e ' + nsha)
         except subprocess.CalledProcessError:
             log.wrn(f"{name_path}: can't get loot; please run "
-                    f'"west update" (need revision {project.revision})')
+                    f'"west update" (no "{n_rev}" ref)')
```

Comment:
as the `n_rev` is taken directly from `refs/heads/manifest-rev`, then how can it be missing ?

Reply:
If the user deleted the manifest-rev branch by hand by accident, for instance.

Reply:
right.